static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,190 @@
+images from a second ZIP file. These compressed archives are downloaded from our
+own
+[iterative/dataset-registry](https://github.com/iterative/dataset-registry)), a
+<abbr>DVC project</abbr> hosted on GitHub. They can be found as
+<abbr>outputs</abbr> of 2 separate
+[DVC-files](/doc/user-guide/dvc-files-and-directories) in the `tutorial/ver`
+directory.
+
+There are a few possible problems with the way this dataset is stored (in 2
+parts, as compressed archives). One is that dataset partitioning is an
+unnecessary complication; It can cause data duplication and other hurdles.
+Another issue is that extra steps are needed to bundle and extract file
+archives. The data compression also raises questions. But most importantly, this
+single dataset is tracked by 2 different DVC-files, instead of 2 versions of the
+same one, as we'll explain next.
+
+Let's download and extract the first archive we discussed above (in an empty
+directory) to visualize the structure of the dataset:
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry \
+          tutorial/ver/data.zip
+$ unzip -q data.zip
+$ tree --filelimit 3
+.
+├── data
+│   ├── train
+│   │   ├── cats [500 entries ...]
+│   │   └── dogs [500 entries ...]
+│   └── validation
+│       ├── cats [400 entries ...]
+│       └── dogs [400 entries ...]
+└── data.zip
+
+7 directories, 1 file
+$ rm -f data.zip
+```
+
+Instead of creating an archive containing the `data/` directory, we can simply
+put it under DVC control as-is! This is done with the `dvc add` command, which
+accepts entire directories. We can then save this first version using Git:
+
+```dvc
+$ git init  # Initialize Git repository
+$ dvc init  # Initialize DVC project
+...
+$ git commit -m "Initialize DVC project"
+...
+$ dvc add data
+Adding 'data' to '.gitignore'...
+Saving information to 'data.dvc'.
+...
+$	git add data.dvc .gitignore
+$ git commit -m "Add 1800 cats and dogs images dataset"
+```
+
+> Refer to
+> [Adding a directory example](/doc/command-reference/add#example-directory) and
+> [Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
+> for more details on what happens under the hood when adding directories.
+
+Let's add the remaining images from the second archive, and see what changes in
+our data directory:
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry \
+          tutorial/ver/new-labels.zip
+$ unzip -q new-labels.zip
+$ tree --filelimit 3
+.
+├── data
+│   ├── train
+│   │   ├── cats [1000 entries ...]
+│   │   └── dogs [1000 entries ...]
+...
+$ rm -f new-labels.zip
+```
+
+An additional 500 images of each kind have been added to the corresponding
+subdirectories, and we probably want to save this update in our
+<abbr>project</abbr>. To do this, we simply run `dvc add` again on the same
+directory, and commit the new version with Git:
+
+```dvc
+$ dvc add data
+Computing md5 for a large number of files. This is only done once.
```

Comment:
let's remove these extra outputs from the docs

Reply:
Removed here. Do you mean all docs? There's one of these in the add cmd ref Directory example.

Reply:
Resolving this since https://github.com/iterative/dvc.org/pull/710#pullrequestreview-303673266 was merged.